app/models/social_share.rb

```diff
@@ -0,0 +1,13 @@
+# == Schema Information
+#
+# Table name: social_shares
+#
+#  id         :bigint(8)        not null, primary key
+#  site_name  :string
+#  created_at :datetime         not null
+#  updated_at :datetime         not null
+#
+class SocialShare < ApplicationRecord
+  SOCIAL_SITES = %w(facebook twitter linkedin)
+  validates :site_name, presence: true, inclusion: { in: SOCIAL_SITES }
```

Comment:
@radinreth please rename site_name to provider_name and SOCIAL_SITES to SOCIAL_PROVIDERS

Reply:
@kakada , please check again bong